frontend/kesaseteli/employer/src/pages/thankyou.tsx

```diff
@@ -27,14 +28,14 @@ const ThankYouPage: NextPage = () => {
     void goToPage('/');
   }, [queryClient, goToPage]);
 
-  if (!applicationId) {
+  if (!isRouterLoading && !applicationId) {
     void goToPage('/');
     return null;
   }
 
   if (applicationQuery.isSuccess) {
     const application = applicationQuery.data;
-    if (application.status === 'draft') {
+    if (applicationId && application.status === 'draft') {
```

Comment:
still could be moved to const enum

Reply:
Perhaps it's just me but I dont like enumerations but prefer the enumerated type instead. Because I think it's more readable and tslint will accept only valid values anyway, also IDE supports auto-suggestions
![Screenshot 2022-02-21 at 10 40 13 AM](https://user-images.githubusercontent.com/7648194/154919177-a6459f70-d626-4587-8e00-e99168fd5843.png)
![Screenshot 2022-02-21 at 10 40 25 AM](https://user-images.githubusercontent.com/7648194/154919179-d5bf4154-8937-4eb6-b103-747d090a5447.png)
![Screenshot 2022-02-21 at 10 41 42 AM](https://user-images.githubusercontent.com/7648194/154919182-c4c3f89c-38fd-417b-87e6-ebc2a3ede94e.png)

Reply:
There is still situations where enums are not redundant: https://stackoverflow.com/questions/40275832/typescript-has-unions-so-are-enums-redundant for example iterating all the possible values, but I havent seen the use case yet in the codebase.